composer.json

```diff
@@ -9,16 +9,18 @@
     "docs": "https://googleapis.github.io/google-auth-library-php/main/"
   },
   "require": {
-    "php": ">=5.4",
+    "php": ">=5.6",
     "firebase/php-jwt": "~2.0|~3.0|~4.0|~5.0",
-    "guzzlehttp/guzzle": "^5.3.1|^6.2.1|^7.0",
-    "guzzlehttp/psr7": "^1.7|^2.0",
+    "guzzlehttp/guzzle": "^6.2.1|^7.0",
+    "guzzlehttp/psr7": "^1.7",
```

Comment:
Dropping `guzzlehttp/psr7:^2.0` ?

Reply:
I'm not sure how that happened - it may have been on accident. I've added it back.